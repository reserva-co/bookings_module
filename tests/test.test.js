import React from 'react';
import { expect } from 'chai';
import { render } from 'enzyme';

import App from './client/components/app.js';

describe('<App />', () => {
  it('renders three `.foo-bar`s', () => {
    const wrapper = render(<App />);
    expect(wrapper.find('.foo-bar')).to.have.lengthOf(3);
  });

  it('renders the title', () => {
    const wrapper = render(<Foo title="unique" />);
    expect(wrapper.text()).to.contain('unique');
  });
});

// import React from 'react';
// import { shallow } from 'enzyme';

// // Components
// import WelcomeMessage from './WelcomeMessage';

// function setup() {
//   const props = {
//     imgPath: 'some/image/path/to/a/mock/image',
//   };
//   const wrapper = shallow(<WelcomeMessage />);
//   return { wrapper, props };
// }

// describe('WelcomeMessage Test Suite', () => {
//   it('Should have an image', () => {
//     const { wrapper } = setup();
//     expect(wrapper.find('img').exists()).toBe(true);
//   });
// });