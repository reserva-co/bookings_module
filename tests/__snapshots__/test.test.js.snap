// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calendar should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
